openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -38,6 +38,13 @@ This document defines a profile of OpenID for Verifiable Credentials in combinat
 
 This document defines a set of requirements for the existing specifications to enable interoperability among Issuers, Wallets and Verifiers of Credentials where a high level of security and privacy is required. This document is an interoperability profile that can be used by implementations in various contexts, be it a certain industry or a certain regulatory environment.
 
+The level of "high assurance" aimed to be achieved in the context of this profile includes the following:
+
+* Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. In scope for this profile are security of the issuance process, protection of issued credentials, and providing mechanisms for the Verifiers to access trustworthy information about the Issuer.
```

Comment:
The final sentence of this paragraph is a bit difficult to read. I think it would be easier to read something like:

```suggestion
* Authenticity of claims: There is strong assurance that the claims within a Credential are valid and bound to the correct Holder. This involves the policies and procedures used to collect and maintain the claims, the authentication of the Holder during issuance, and the protection of claim authenticity both at rest (in the wallet) and during presentation. The scope for this profile is: security of the issuance process, protection of issued credentials, and mechanisms for the Verifiers to access trustworthy information about the Issuer.
```